### DISCUSSION TITLE
Don't get CourseRegistration log in Seq

### DISCUSSION THREAD

**Discussion by mahedee** [OpenCodeFoundation/eSchool#355] [Q&A] [answered]
I have tried to implement serilog in CourseRegistration services. But It doesn't show in http://localhost:5140/#/events . Here is my code. Any one help me please. What is the wrong with my code?

https://github.com/mahedee/eSchool/tree/feature-course-registration

@ratanparai

**Comment by ratanparai** [comment]
I cloned your repo and run the application in my machine and `CourseRegistration` service is writing logs to `Seq`. 

<img width="1092" alt="Screenshot 2021-04-09 at 9 49 59 PM" src="https://user-images.githubusercontent.com/1322068/114206818-89fdb780-997d-11eb-9134-abc1c1715b34.png">

Can you please double check on your end?

**Reply by mahedee** [reply]
Still same problem. Don't find log. Which url have you checked? @ratanparai

**Comment by mahedee** [comment, marked as answer]
Solved: Run application from visual studio 2019. Make docker-compose as stratup project. Then run docker compose. Now check the url: http://localhost:5140/#/events
